content/sse/index.textile

```diff
@@ -136,7 +132,7 @@ h3(#event-stream). Plain HTTP event stream
 
 h6. GET realtime.ably.io/event-stream
 
-Start a streaming HTTP request, for ease of consuming with any HTTP library that supports streaming. When possible, we recommend using an SSE library as opposed to a raw HTTP stream as SSE libraries automatically handle reconnecting and resuming from the last received ID.
+Start a streaming HTTP request, for ease of consuming with any HTTP library that supports streaming. Instead of SSE events, uses JSON envelopes. When available, we recommend using an SSE library as opposed to a raw HTTP stream as SSE libraries automatically handle reconnecting and resuming from the last received ID.
```

Comment:
I'd suggest something as follows for the first two sentences:

`Starts a streaming HTTP request. This allows for messages to be easily consumed with any HTTP library that supports streaming. This uses JSON envelopes instead of SSE events....`